modules/nw-sriov-hwol-improving-network-traffic-performance.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-hardware-offloading.adoc
+
+:_content-type: PROCEDURE
+[id="improving-network-traffic-performance-using-vf_{context}"]
+= Improving network traffic performance using a virtual function
+
+Follow this procedure to assign a virtual function to the OVN-Kubernetes management port and increase its network traffic performance.
+
+This procedure results in the creation of two pools: the first has a virtual function used by OVN-Kubernetes, and the second comprises the remaining virtual functions.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Add the `network.operator.openshift.io/smart-nic` label to each worker node with a SmartNIC present by running the following command:
++
+[source,terminal]
+----
+$ oc label node <node-name> network.operator.openshift.io/smart-nic=
+----
++
+Use the `oc get nodes` command to get a list of the available nodes.
```

Comment:
I feel like this should be either the first step of the procedure or in a `[TIP]` block. It's awkward (to me) as a command beneath a command in a code block.

Reply:
I thought of including it as the first step as well, but it may not be _necessary_. Honestly, I'm not certain. Using a `[TIP]` admonition sounds reasonable to me.